### DISCUSSION TITLE
retrospect-2023-1st/

### DISCUSSION THREAD

**Discussion by giscus** [koderpark/BlogComment#4] [Announcements]
# retrospect-2023-1st/

국민대학교 진학 후 첫 학기가 끝나고

본가로 돌아왔다.

올해 상반기는 기쁜일도 많았고, 좀 아쉬운 점도 많았다.

이번 분기에서 좀 아쉬웠던 부분들 보완해서 하반기때는 좀 더 개선을 도모해보고자 한다

학업

대학교의 학업 관련해서는 아주 좋았다고 말할 수 있다.

아무래도 특기자전형으로 입학하였다 보니 수시,정시로 들어온 학생들에 비해

고등학교 과목에 대한 공부가

https://blog.koder.page/retrospect-2023-1st/

**Comment by ljlljlll** [comment]
Fan 이에요 :D
